airbyte-integrations/connectors/destination-langchain/destination_langchain/config.py

```diff
@@ -0,0 +1,123 @@
+#
+# Copyright (c) 2023 Airbyte, Inc., all rights reserved.
+#
+
+import json
+import re
+from typing import List, Literal, Optional, Union
+
+from jsonschema import RefResolver
+from pydantic import BaseModel, Field
+
+
+class ProcessingConfigModel(BaseModel):
+    chunk_size: int = Field(
+        ...,
+        title="Chunk size",
+        maximum=8191,
+        description="Size of chunks in tokens to store in vector store (make sure it is not too big for the context if your LLM)",
+    )
+    chunk_overlap: int = Field(
+        title="Chunk overlap",
+        description="Size of overlap between chunks in tokens to store in vector store to better capture relevant context",
+        default=0,
+    )
+    text_fields: Optional[List[str]] = Field(
```

Comment:
it feels wrong to configure the text fields at the destination config level because it means it cannot be reused across connections.

Maybe this belongs in a connection config alongside the configured catalog? Or maybe the source knows which fields are text fields

Reply:
I agree that would be ideal, but it's a much bigger lift as it would require changes to the platform in some way or another. This is to get a starting point. You can reuse it across connections as non-existing fields will be ignored, so you can set it to the total set of fields that are relevant for texts for all sources.

I agree that this isn't how it should work, this should be part of the connection, it's like reverse ETL in this sense. I hope we can start to get deeper into this.